.zuul.yaml

```diff
@@ -17,8 +17,8 @@
       nodes:
         - name: ci-node-32
           label: cloud-fedora-32-small
-    pre-run: playbooks/fedora-32/setup-env.yaml
-    run: playbooks/fedora-32/system-test-fedora-32.yaml
+    pre-run: playbooks/setup-env.yaml
+    run: playbooks/system-test.yaml
```

Comment:
Right now there are no package conflicts and the `setup-test-{fedora-version}.yml` are identical. I think that it can be a source of future issues (not updating all the files), also unnecessary duplication.
If some RPM packages change in the future, they can be manage easily using variables inside the playbook, as there will be just a few, but the rest of the playbook will be identical.

Reply:
Ok, then. :)

Just note that I know very little about Ansible, and even less about using variables, etc.. So, I might end up having to ask for your help.

Reply:
Although, we should probably still retain the `fedora` sub-directory, because we might be adding support for other distributions like Ubuntu.